### DISCUSSION TITLE
Minio Drives and FSTAB: UUIDs vs Sequential Lables

### DISCUSSION THREAD

**Discussion by andar1an** [minio/minio#18843] [Q&A] [answered]
Hello, 

I am referencing these docs:
https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.html#id2

They state that minio "requires" sequential drive numbers for parameter expansion. Does this mean I am required to use labels, and can't mount by UUID? 

Currently I mount by UUID, and if it is possible to continue to do that, it would be preferred. If not I can change to using labels. 

Help to understand would be appreciated, thank you.

**Comment by harshavardhana** [comment, marked as answer]
You can mount via UUID or drive labels. We recommend labels because it's human friendly.

You can use what you are comfortable with .
